packages/run-protocol/test/supports.js

```diff
@@ -26,4 +43,77 @@ export function makeFakeInnerVault(
   });
   // @ts-expect-error pretend this is compatible with VaultKit
   return vault;
-}
+};
+
+/**
+ *
+ * @param {string} sourceRoot
+ * @returns {Promise<SourceBundle>}
+ */
+export const makeBundle = async sourceRoot => {
+  const url = await importMetaResolve(sourceRoot, import.meta.url);
```

Comment:
this is a little awkward in that the `sourceRoot` has to be relative to this file, not relative to the file in which it occurs.

Reply:
That surprised me, but turned out well because then everything in test was relative to the test directory. Since various projects were reaching into other projects for files, this seemed good. We could have a shared helper that you then wrap inside a per-component test directory.  Which piece is the magic?  `importMetaResolve` of `import.meta.url`?

Reply:
I have a mild preference for using absolute paths for anything that goes between modules. I tend to open-code `new URL('../xyz'.js, import.meta.url).pathname`.

But I don't feel strongly about it. I can go with...

> everything in test was relative to the test directory